### DISCUSSION TITLE
Enhancement: Saving preferences

### DISCUSSION THREAD

**Discussion by JackOfAll64Trades** [FunkinCrew/Funkin#1654] [Ideas]
I was wondering. When you start up the game, you can set your preferences. **But it's really annoying that those preferences don't save for the next time you play.** If there is someway to _save your preferences_, like your scores, to your Newgrounds account, that would be greatly appreciated.

**Comment by actuallyasmartname** [comment]
This would be a very nice feature to see :thumbsup:

**Comment by Windows1Door2Daniel** [comment]
i think the cookies was deleted

because i play on newgrounds without account and all my save data is still undeleted
